dir.props

```diff
@@ -452,6 +453,12 @@
         <NuGetTargetMoniker>.NETCoreApp,Version=v1.0</NuGetTargetMoniker>
       </PropertyGroup>
     </When>
+    <When Condition="'$(TargetGroup)'=='netcoreapp1.1'">
```

Comment:
we could (and eventually should), but probably in a different PR as there are some other places we would need to add netstandard1.7 entries as well. afaik we're not using netstandard1.7 anywhere in CoreFX yet.